First draft, need review to see if there is missing information, or s… (#2839)
CSCfi/csc-user-guide: docs/cloud/rahti/tutorials/highavailability.md

     
     For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/.

     * With parameters:
        * Memory Limit=512Mi
        * Namespace=openshift
        * Database Service Name=db
        * PostgreSQL Connection Username=myuser
        * PostgreSQL Connection Password=mypassword
        * PostgreSQL Database Name=mydb
        * Version of PostgreSQL Image=10-el8

--> Creating resources ...
    secret "db" created
    service "db" created
Warning: apps.openshift.io/v1 DeploymentConfig is deprecated in v4.14+, unavailable in v4.10000+
    deploymentconfig.apps.openshift.io "db" created
--> Success
    Application is not exposed. You can expose services to the outside world by executing one or more of the commands below:
     'oc expose service/db' 
    Run 'oc status' to view your app.
```

The template "PostgreSQL Ephemeral" uses memory to store the data, and it is used only for testing as when the Pod of the database is restarted the data is lost. If you want to have a proper production database, we recommend you use [Pukki DBaaS](../../dbaas/index.md).

> After waiting for some time, the `rahti-ha-tutorial` Pod should restart by itself and the error will be fixed:

Reviewer: ```suggestion
After a few minutes, the `rahti-ha-tutorial` Pod should restart by itself and the error will be fixed:
```
